docs/deploy-for-ocr.md

```diff
@@ -3,48 +3,102 @@
 ```
 git clone https://github.com/SuSy-One/susy-chainlink-auditor
 cd susy-chainlink-auditor/
-mkdir chainlink_data
-cd chainlink_data
+mkdir chainlink-1
+cd chainlink-1
 ```
 ## Access settings
 
 Please note that Chainlink node enforces a password policy that requires a password to have at least 3 numbers and 3 symbols.
-First, create two config files in `./chainlink_data`:
+First, create two config files in folder `./chainlink-1`:
 
-`.password` -- node access password
+`.password` -- node access password (of your choosing):
 ```
 echo "MyVeryStrongP@ssword123" > .password
 ```
 
-`.api` - login/password for Chainlink Operator GUI access
+`.api` -- login/password for Chainlink Operator GUI access (both of your choosing, used only locally):
+```
+touch .api // Crating the file
+nano .api // Opening editor
```

Comment:
This seems to me a too specific recommendation because operators may use different environments